### ISSUE TITLE
decorators.py 

### ISSUE THREAD

**Issue by powerhouseofthecell** [rdegges/django-twilio#102]
I'm under the impression that I must be missing something important here, because I can't seem to find this issue elsewhere on the web. However, my pip-installed version of django_twilio has a decorators.py file which is inconsistent with the one here on GitHub. The main problem seems to arise from these two import statements: 
```python
from twilio.twiml import Verb
from twilio.util import RequestValidator
```
The above statements should be:
```python
from twilio.twiml import TwiML as Verb
from twilio.request_validator import RequestValidator
```
The current GitHub version has this correction, but the pip-installed version does not. 

Please do let me know if I am foolishly missing something, or if this is a known problem. 

Best,
potc

**Comment by mulka** [comment]
@rdegges probably needs to push a new version to pypi.

**Comment by audrummer15** [comment]
Definitely need to get this out to pypi asap. I'm unable to use the default pip installations of twilio and django_twilio.

Awesome work though on this project guys! And thanks for posting @powerhouseofthecell

**Comment by willex** [comment]
Having the same issue! Is there an ETA on when this is going to get pushed? I also can't install through pip because of this!

Thanks for the post @powerhouseofthecell!

**Comment by audrummer15** [comment]
Unfortunately I think the repo is entering an unmaintained state. @willex your best bet is to probably create your own fork and use that.

**Comment by charlietw** [comment]
+1 for pushing this amendment to pypi

**Comment by mulka** [comment]
I've pushed a forked version of django-twilio to pypi as django-twilio2 which has the fix for this

**Comment by rdegges** [comment]
Hey @mulka -- I'm happy to add you as a owner of the repo here. Your work would be greatly appreciated!

**Comment by rdegges** [comment]
If you're amenable to that I'm happy to hand over commit and PyPI access. I've mentioned this before in different issues, but I no longer use Django in my day-to-day work, and haven't been keeping up with changes to these libraries.

**Comment by jheld** [comment]
Hi all,

I'm a new contributor/maintainer of this project. Thanks @rdegges. I'll be looking through the issues and PRs as quickly as I can. Please provide any updates (usage, issues, workarounds) that you may have, or patches, as PRs; otherwise it's in order of importance, followed by morality.

**Comment by jheld** [comment]
Just to update everyone here, `0.9.0a1` is available on PyPI. It has all of the support for twilio 6.x and support for django 2.0 as is based on the MRs and issues here so far.

My company is testing today and deploying tonight with the alpha build from PyPI. If all goes well in the coming days up through the sprint (maybe earlier), I'll release either a `rc` build or just the proper `0.9.0` build to PyPI.

Please let me know if you need anything.

**Comment by jheld** [comment]
I'm closing this issue for now. If you still have issues or any comments related to this issue, please re-open as needed.

**Comment by jheld** [comment]
Hey guys, I have released version `0.9.0`, so if you're still receiving notifications for github, hopefully this reaches you!
